scripts/lib/api/releaseNotes.ts

```diff
@@ -108,12 +108,17 @@ export async function updateHistoricalTocFiles(pkg: Pkg): Promise<void> {
   ).filter((file) => file.isDirectory() && file.name.match(/[0-9].*/));
 
   for (let folder of historicalFolders) {
-    let tocFile = await readFile(
-      `${getRoot()}/docs/api/${pkg.name}/${folder.name}/_toc.json`,
-      {
-        encoding: "utf8",
-      },
-    );
+    const tocFilePath = `${getRoot()}/docs/api/${pkg.name}/${
```

Comment:
Why this code? I would expect `_toc.json` to always be defined.

Reply:
> Handle empty historical version folders

Oh, try running `git clean -fd`.

Reply:
> Do you know if it was deleted manually or via the script?

The script deleted it then errored due to the empty folder so didn't recreate it.

> Oh, try running git clean -fd.

I've deleted the empty folder now I know that's what caused it, this fix is to prevent anyone else tripping over the same problem.

Reply:
I don't think it's worth the code complexity. Generally I _would_ want us to error if we had empty folders

Reply:
Possibly because I generated the docs with the historical flag at some point? Honestly not sure; I can remove it if you prefer.

Reply:
Ah, yeah possibly. I'd prefer to not have it because it's extra code complexity and I _would_ want us to error if we had a malformed historical folder.